xbmc/LangInfo.cpp

```diff
@@ -1133,6 +1134,11 @@ void CLangInfo::SettingOptionsStreamLanguagesFiller(const CSetting *setting, std
 
   std::string dummy;
   SettingOptionsISO6391LanguagesFiller(NULL, list, dummy, NULL);
+  SettingOptionsLanguageNamesFiller(NULL, list, dummy, NULL);
+  std::set<std::pair<std::string, std::string>> languages;
+  unsigned size = list.size();
+  for (unsigned i = 0; i < size; ++i) languages.insert(list[i]);
```

Comment:
* Something odd happpened here with indentation.
* `std::set::insert` also has a range version. Otherwise using auto range would also be preferred over the manual for loop.
* would be nice if the resulting list could be sorted (see `SettingOptionsLanguageNamesFiller()` for that).